README.md

```diff
@@ -1,4 +1,4 @@
-[![CI Status](https://github.com/AY2425S1-CS2103-F13-1/tp/workflows/Java%20CI/badge.svg)](https://github.com/AY2425S1-CS2103-F13-1/tp/actions)
+[![CI Status](https://github.com/AY2425S1-CS2103-F13-1/tp/actions/workflows/gradle.yml/badge.svg?branch=master)](https://github.com/AY2425S1-CS2103-F13-1/tp/actions)
```

Comment:
For some reason, the branch query doesn't work without the `actions` path in the URL...